lib/magma/attributes/child.rb

```diff
@@ -1,14 +1,14 @@
 class Magma
   class ChildAttribute < Attribute
     include Magma::Link
-    def json_for record
-      record[name]
+    def query_to_payload(value)
+      value
     end
 
-    def update_record record, link
-      link_model.update_or_create(link_model.identity => link) do |obj|
-        obj[ self_id ] = record.id
-      end
+    def revision_to_record(record, value)
+    end
```

Comment:
Are these supposed to be abstract methods (subclasses should impelement)?
If so, might be useful to indicate this with a comment or a `raise NotImplementedError`

Reply:
There is an implicit nil which I think gets compacted away.